src/libraries/System.Formats.Tar/tests/TarReader/TarReader.File.Tests.cs

```diff
@@ -556,6 +556,21 @@ public void Read_Archive_LongPath_Over255(TarEntryFormat format, TestTarFormat t
             Assert.Null(reader.GetNextEntry());
         }
 
+        [Fact]
+        public void ReadEntriesWithSlashDotPrefix()
```

Comment:
Does this test provide any value over ExtractEntriesWithSlashDotPrefix? It appears to be a subset of that test.

Reply:
I guess not. I wrote this test first, to make sure my fix was working and not throwing when an entry was _only_ read.

Then I wrote the other one, which would not just read, but also would extract the entry, and that's where the exception was originally happening.

I'll remove this test.